packages/itwinui-react/src/core/utils/functions/dev.ts

```diff
@@ -3,7 +3,9 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 
-const isJest = typeof (globalThis as any).jest !== 'undefined';
+// @ts-expect-error -- jest is not used in iTwinUI but may be used in consuming apps
+const isJest = typeof jest !== 'undefined';
+
```

Comment:
nit: Just noticed an extra line (unless it's intentional). If we work on this file later, maybe we can remove this line.

Reply:
It's intentional, since there is a comment above. The newline indicates (at least to me) that the comment does not relate to the code below the newline.